src/SFA.DAS.Campaign.Web/Views/Shared/Components/Form/Register-Thanks.cshtml

```diff
@@ -4,5 +4,5 @@
 <p>Keep an eye on your inbox for all details of future updates to the site.</p>
 <a role="button" class="button button--with-icon button--success" href="#" id="btn-register-interest-success">
     <img src="/campaign/images/icon-cross-white.svg" class="button__icon button__icon--close" alt="" />
-    <span class="button__text">Close</span>
+    <span class="button__text">Close <span>the register interest popup</span></span>
```

Comment:
should this have a visually hidden class on it?

Reply:
Good spot, added now thanks